src/tools/db-upgrade/DbUpgraderTool.cpp

```diff
@@ -1,15 +1,148 @@
-/* Copyright (c) 2020 vesoft inc. All rights reserved.
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
  *
  * This source code is licensed under Apache 2.0 License,
  * attached with Common Clause Condition 1.0, found in the LICENSES directory.
  */
 
 #include "common/base/Base.h"
-#include "utils/NebulaKeyUtils.h"
-#include "tools/db-upgrade/NebulaKeyUtilsV1.h"
+#include "tools/db-upgrade/DbUpgrader.h"
+
+
+void printHelp() {
+    fprintf(stderr,
+           R"(  ./db_upgrade --src_db_path=<path to rocksdb> --dst_db_path=<path to rocksdb>  --upgrade_meta_server=<ip:port,...>
+
+required:
+       --src_db_path=<path to rocksdb>
+         Source data path(data_path in storage 1.0 conf) to the rocksdb data directory,
+         multi paths should be split by comma.
+         If nebula 1.0 was installed in /usr/local/nebula,
+         the db_path would be /usr/local/nebula/data/storage
+         note: db_path not /usr/local/nebula/data/storage/nebula
+         Default: ""
+
+       --dst_db_path=<path to rocksdb>
+         Destination data path(data_path in storage 1.0 conf) to the rocksdb data directory,
+         multi paths should be split by comma.
+         If nebula 2.0  was installed in /usr/local/nebula,
```

Comment:
changed

Reply:
I don't know how to write this description yet. Because if we specify both `src_db_path` and `dst_db_path` as `/usr/local/nebula/data/storage`, will the tool work?

So we'd better tell the user two paths must be different. I hope Mr Target can run this tool with the help. 😆

Reply:
add a note to explain